pkg/client/unversioned/pods_test.go

```diff
@@ -194,22 +195,22 @@ func TestPodGetLogs(t *testing.T) {
 		Follow:     true,
 		Timestamps: true,
 	}
-	c := &testClient{}
-
-	request := c.Setup(t).Pods(ns).GetLogs("podName", opts)
-	if request.verb != "GET" {
-		t.Fatalf("unexpected verb %q, expected %q", request.verb, "GET")
-	}
-	if request.resource != "pods" {
-		t.Fatalf("unexpected resource %q, expected %q", request.subresource, "pods")
-	}
-	if request.subresource != "log" {
-		t.Fatalf("unexpected subresource %q, expected %q", request.subresource, "log")
+	c := &testClient{
+		Request: testRequest{
+			Method: "GET",
+			Path:   testapi.Default.ResourcePath("pods", ns, "podName") + "/log",
+			Query: url.Values{
+				"follow":     []string{"true"},
+				"timestamps": []string{"true"},
+			},
+		},
+		Response: Response{StatusCode: http.StatusOK},
 	}
-	expected := map[string]string{"container": "", "follow": "true", "previous": "false", "timestamps": "true"}
```

Comment:
Who's checking this after the change? ValidateCommon seems not checking the request.params.